package.json

```diff
@@ -71,5 +71,10 @@
     "budgetPercentIncreaseRed": 20,
     "minimumChangeThreshold": 0,
     "showDetails": true
+  },
+  "pnpm": {
+    "overrides": {
+      "compression": "1.8.1"
```

Comment:
This isn't a good way to solve this issue in my opinion. A dependency that we use decided to use a certain compression version and they tested with this version.
By overwriting this we may break the functionality now or later in an uncontrolled way.
instead of doing overwrites we should check if there is a new version of the package using compression and if they updated their dependency.